src/eureka/lib/meanerr.py

```diff
@@ -0,0 +1,106 @@
+import numpy as np
+
+
+def meanerr(data, derr, mask=None, err=False, status=False):
+    """
+    Calculate the error-weighted mean and the error in the
+    error-weighted mean of the input data, omitting masked data, NaN
+    data or errors, and data whose errors are zero.
+
+    Parameters
+    ----------
+    data:   ndarray
+            The data to average.
+    derr:   ndarray
+            The 1-sigma uncertainties in data, same shape as data.
+    mask:   ndarray
+            A 1 indicates the corresponding element of Data is good, a
+            0 indicates it is bad, same shape as data.
+    err:    boolean
+            Set to True to return error in the mean.
+    status: boolean
+            Set to True to return a bit flag.  If value=0, result is good.
+            Bits: 0 = NaN(s) in data.
+            1 = some errors equal zero.
+            2 = masked pixel(s) in data.
+
+    Return
+    ------
+    This function returns the error-weighted mean of the unmasked
+    elements of Data. If err or status were set to True, return a
+    tuple.
```

Comment:
Needs to be "Returns" to be put in the docs. Also please format in the same way as the rest of the package (an example is below):

```
Returns
-------
dtype1
    Description of variable1
dtype2
    Description of variable2
```